packages/amp/src/__tests__/amp.server.test.ts

```diff
@@ -0,0 +1,37 @@
+import { createStore, InitializedStore } from "@frontity/connect";
+import clone from "clone-deep";
+import { Packages } from "../../types";
+import Amp from "../server";
+
+describe("AMP server tests", () => {
+  let store: InitializedStore<Packages>;
+
+  beforeEach(() => {
+    // Initialize the store
+    store = createStore<Packages>(clone(Amp));
+
+    // We have to mock these since the `createStore` it's not the one used
+    // on the server and client.
+    store.libraries = {
+      source: {
+        parse: jest.fn(),
+        normalize: jest.fn(),
+        stringify: jest.fn(),
+      },
+      frontity: {
+        render: jest.fn(),
+        template: jest.fn(),
+      },
+    };
+  });
+
+  test("The initial values are overwritten with the Amp ones", () => {
+    const previousRender = store.libraries.frontity.render;
+    const previousTemplate = store.libraries.frontity.template;
+
+    store.actions.amp.beforeSSR();
+
```

Comment:
Same as for the `client.tests.ts`.

Do you think we could add a snapshot test for `render` and `template` ?

```suggestion
expect(store.libraries.frontity.render.toString()).toMatchInlineSnapshot();
expect(store.libraries.frontity.render.toString()).toMatchInlineSnapshot();
```

Reply:
Indeed we can! Will update the tests. Thank you!